app.ts

```diff
@@ -0,0 +1,233 @@
+let headingColumns: any = document.querySelector("#column-headings-container"); //Headings;
+let infoColumns: any = document.querySelector("#info-columns-container"); //Information;
```

Comment:
Add whitespace between // and comment. Can probably remove the semi-colon as well.